src/StartJourneyButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './App.css';
+
+type Props = {
+    gotStarted: boolean,
+    setGotStarted: Function
+};
+
+function StartJourneyButton(props: Props) {
+    function handleClick() {
+        props.setGotStarted(!props.gotStarted);
```

Comment:
```suggestion
        setGotStarted(!gotStarted);
```

but I would say we need just to set the variable to `true` here
```suggestion
        setGotStarted(true);
```